cmd/lotus-storage-miner/actor.go

```diff
@@ -698,3 +700,221 @@ var actorSetOwnerCmd = &cli.Command{
 		return nil
 	},
 }
+
+var actorProposeChangeWorker = &cli.Command{
+	Name:      "propose-change-worker",
+	Usage:     "Propose a worker address change",
+	ArgsUsage: "[address]",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "really-do-it",
+			Usage: "Actually send transaction performing the action",
+			Value: false,
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		if !cctx.Bool("really-do-it") {
+			fmt.Fprintln(cctx.App.Writer, "Pass --really-do-it to actually execute this action")
+			return nil
+		}
```

Comment:
I'd put this after sanity checks, so running this command without that flag is actually useful